ops_reactive_interface.py

```diff
@@ -21,17 +27,21 @@
 from charmhelpers.core import hookenv
 from charmhelpers.core import unitdata
 
-try:
-    from importlib.metadata import entry_points
-except ImportError:
-    from pkg_resources import iter_entry_points
+from pkg_resources import iter_entry_points
 
-    def entry_points():
-        eps = {}
-        for role in ('provides', 'requires', 'peers'):
-            group = 'ops_reactive_interface.' + role
-            eps[group] = list(iter_entry_points(group))
-        return eps
```

Comment:
This fixes the entry_points usage as prescribed [here][] by @wolsen 

[here]: https://github.com/canonical/ops-reactive-interface/compare/main...wolsen:ops-reactive-interface:fix-endpoints